packages/ember-views/tests/utils/helpers.js

```diff
@@ -0,0 +1,10 @@
+import isEnabled from 'ember-metal/features';
+import require from 'require';
+
+export let OutletView;
+
+if (isEnabled('ember-glimmer')) {
+  OutletView = require('ember-glimmer/views/outlet').default;
+} else {
+  OutletView = require('ember-htmlbars/views/outlet').OutletView;
```

Comment:
nevermind, I thought it did, but I was wrong. I'm trying to get away from ember-views requiring ember-glimmer or ember-htmlbars, I'm trying to have it just be where stuff that is shared not branched is.